nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_umac_if.h

```diff
@@ -669,6 +670,8 @@ struct nrf_wifi_channel {
 struct nrf_wifi_scan_params {
 	/** If 0x1, RPU force passive scan on all channels */
 	unsigned short passive_scan;
+	/** If true, skip local and IANA Unicast reserved MACs **/
+	unsigned char skip_local_mac;
```

Comment:
This might be slightly misleading that `skip local interface MAC`, can we please rename to `skip_colocated_aps` or `skip_local_admin_macs`?

Reply:
Will go with skip_local_admin_macs

Reply:
Done